src/main/java/us/kilroyrobotics/RobotContainer.java

```diff
@@ -78,57 +103,75 @@ private void configureBindings() {
                         // negative X (left)
                         ));
 
-        joystick.a().whileTrue(drivetrain.applyRequest(() -> brake));
-        joystick.b()
+        driverController.a().whileTrue(drivetrain.applyRequest(() -> brake));
+        driverController
+                .b()
                 .whileTrue(
                         drivetrain.applyRequest(
                                 () ->
                                         point.withModuleDirection(
                                                 new Rotation2d(
-                                                        -joystick.getLeftY(),
-                                                        -joystick.getLeftX()))));
+                                                        -driverController.getLeftY(),
+                                                        -driverController.getLeftX()))));
 
-        joystick.pov(0)
+        driverController
+                .pov(0)
                 .whileTrue(
                         drivetrain.applyRequest(
                                 () -> forwardStraight.withVelocityX(0.5).withVelocityY(0)));
-        joystick.pov(180)
+        driverController
+                .pov(180)
                 .whileTrue(
                         drivetrain.applyRequest(
                                 () -> forwardStraight.withVelocityX(-0.5).withVelocityY(0)));
 
         // Run SysId routines when holding back/start and X/Y.
         // Note that each routine should be run exactly once in a single log.
-        joystick.back().and(joystick.y()).whileTrue(drivetrain.sysIdDynamic(Direction.kForward));
-        joystick.back().and(joystick.x()).whileTrue(drivetrain.sysIdDynamic(Direction.kReverse));
-        joystick.start()
-                .and(joystick.y())
+        driverController
+                .back()
+                .and(driverController.y())
+                .whileTrue(drivetrain.sysIdDynamic(Direction.kForward));
+        driverController
+                .back()
+                .and(driverController.x())
+                .whileTrue(drivetrain.sysIdDynamic(Direction.kReverse));
+        driverController
+                .start()
+                .and(driverController.y())
                 .whileTrue(drivetrain.sysIdQuasistatic(Direction.kForward));
-        joystick.start()
-                .and(joystick.x())
+        driverController
+                .start()
+                .and(driverController.x())
                 .whileTrue(drivetrain.sysIdQuasistatic(Direction.kReverse));
 
         // Reset the field-centric heading on left bumper press
-        joystick.leftBumper().onTrue(drivetrain.runOnce(() -> drivetrain.seedFieldCentric()));
+        driverController
+                .leftBumper()
+                .onTrue(drivetrain.runOnce(() -> drivetrain.seedFieldCentric()));
 
         // Toggle between high and low speeds
-        joystick.x().onTrue(toggleMaxSpeed);
+        driverController.x().onTrue(toggleMaxSpeed);
```

Comment:
This and the following lines need to follow the formatting of the above lines

Reply:
I don't like that kind of formatting, but if you and colin are ok with it, so am i.

Reply:
Formatting was actually wrong so this should be fixed.